src/Catalyst.Core.Lib.Tests/UnitTests/Validators/TransactionValidatorTests.cs

```diff
@@ -160,15 +147,31 @@ public FakeContext(ISignature signature, bool verifyResult)
             public int SignatureLength { get; }
             public int SignatureContextMaxLength { get; }
             public IPrivateKey GeneratePrivateKey() { throw new NotImplementedException(); }
-            public IPublicKey GetPublicKeyFromPrivateKey(IPrivateKey privateKey) { throw new NotImplementedException(); }
+
+            public IPublicKey GetPublicKeyFromPrivateKey(IPrivateKey privateKey)
+            {
+                throw new NotImplementedException();
+            }
+
             public IPublicKey GetPublicKeyFromBytes(byte[] publicKeyBytes) { throw new NotImplementedException(); }
             public IPrivateKey GetPrivateKeyFromBytes(byte[] privateKeyBytes) { throw new NotImplementedException(); }
             public byte[] ExportPrivateKey(IPrivateKey privateKey) { throw new NotImplementedException(); }
             public byte[] ExportPublicKey(IPublicKey publicKey) { throw new NotImplementedException(); }
-            
-            public ISignature Sign(IPrivateKey privateKey, ReadOnlySpan<byte> message, ReadOnlySpan<byte> context) => _signature;
-            public ISignature GetSignatureFromBytes(byte[] signatureBytes, byte[] publicKeyBytes) => Substitute.For<ISignature>();
-            public bool Verify(ISignature signature, ReadOnlySpan<byte> message, ReadOnlySpan<byte> context) => _verifyResult;
+
+            public ISignature Sign(IPrivateKey privateKey, ReadOnlySpan<byte> message, ReadOnlySpan<byte> context)
+            {
+                return _signature;
+            }
+
+            public ISignature GetSignatureFromBytes(byte[] signatureBytes, byte[] publicKeyBytes)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'GetSignatureFromBytes'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)